contract_manager/src/base.ts

```diff
@@ -20,6 +20,18 @@ export abstract class Storable {
   abstract toJson(): any;
 }
 
+export interface Price {
+  price: string;
+  conf: string;
+  publishTime: string;
+  expo: string;
+}
+
+export interface PriceFeed {
+  price: Price;
+  emaPrice: Price;
+}
+
```

Comment:
Hmmm, we have a PriceFeed definition in price-service-sdk. I think it's good to have it as it is now and don't using it. Just saying it for awareness.